b2b-sample/guardio-insurance-sample-application/components/editUser/editUserComponent.js

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useState } from 'react';
```

Comment:
Leave a line after the license header. Check other places also.

Reply:
This was after adding the eslint in the `identity-apps` to the project.

### Commit
- https://github.com/wso2/samples-is/commit/e50bfd37f9fdd026c3f685b6fe602f7fdcc542e8#diff-6e806d4538a673d7ea75a914962e3f09e15ecbb1754339a574f54c5c619636d7